lib/pages/contacts_tab/contacts_tab_body_view.dart

```diff
@@ -572,6 +577,11 @@ class _Contact extends StatelessWidget {
         contact: contact,
         highlightKeyword: controller.textEditingController.text,
         enableInvitation: true,
+        onContactTap: () => controller.onRecentContactTap(
```

Comment:
Should change the name of `controller.onRecentContactTap` -> `controller.onContactTap`